src/components/CreateInstructorPage/CreateInstructorForm.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { Field, FieldArray, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { push } from 'connected-react-router';
+
+import AreasOfExpertise from './AreasOfExpertise';
+import SocialLinks from './SocialLinks';
+import ImageUpload from '../../components/ImageUpload';
+import RenderInputTextField from '../RenderInputTextField';
+import RenderSelectField from '../RenderSelectField';
+import RichEditor from '../../components/RichEditor';
+import store from '../../data/store';
+
+
+const basicValidate = value => (value ? undefined : 'This field is required');
+
+const extractOrgChoices = (instructorOptions) => {
+  const { data = {} } = instructorOptions;
+
+  if (!data.actions) {
+    return [];
+  }
+
+  const { choices = [] } = data.actions.POST.position.children.organization;
+
+  const defaultOption = [{ label: 'Select instructor organization', value: '' }];
+  return defaultOption.concat(choices
+    .map(choice => ({ label: choice.display_name, value: choice.value })));
+};
+
+const BaseCreateInstructorForm = ({
+  handleSubmit,
+  pristine,
+  submitting,
+  isCreating,
+  instructorOptions,
+}) => {
+  const formControlDisabled = pristine || submitting || isCreating;
+
+  return (
+    <React.Fragment>
+      <div className="create-instructor-form row">
+        <div className="col">
+          <form onSubmit={handleSubmit}>
+            <Field
+              name="image"
+              component={ImageUpload}
+              label={<React.Fragment>Image: <span className="required">*</span></React.Fragment>}
+              id="image"
+              required
+            />
+            <Field
+              name="firstName"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <React.Fragment>
+                  First name <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="lastName"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <React.Fragment>
+                  Last name <span className="required">*</span>
+                </React.Fragment>
+              }
+              required
+            />
+            <Field
+              name="title"
+              component={RenderInputTextField}
+              type="text"
+              label={
+                <React.Fragment>
+                  Title <span className="required">*</span>
```

Comment:
Can we make this Position Title or just Position or something that nudges people to not enter "Dr." here?

Reply:
Changed